### ISSUE TITLE
--no-semi confused by class properties named `get` and `set`

### ISSUE THREAD

**Issue by levithomason** [prettier/prettier#3431]
**Prettier 1.9.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAucAbAhgZ0wAgOJwwDKhAggE7kQDuuwAOlDjgOaE4C8OAFAJScA+HAHI2MYY2aZ2XPoJHTxjSTigQYBGJx78OQ4Wq1iJTVepJbZu-YZyKTAX0YgANCAgAHGAEtomZKDolDQACkEI-ijoAG4Q3gAmriBYMMgAZuio0m4ARuToYADWhEQeBd5QLMgw5ACucG7xEGDpmdkgFdLkMCH5LAC26K1ZDSAAVpgAHgBC+UUl6P1wADIVcMPtELUwHtsATBujZeRdyCA56DkAnqjQSR7kFTAA6gkwABbIABwADG4PEGkz3yHjODzgXWi6zc5DgAEdat5Yb10AMhkgMiM3NJ+t5DtiKixUHAAIq1dTrDFtUYwS6veIfZB7Nw1dDeVCEgDCEH6gzOaig0JAtWkABVLpFMdIHA4gA)
```sh
--no-semi
```

**Input:**
```jsx
class GetSetArrows {
  get = () => 'get'
  set = () => 'set'

  notGet = () => 'not get'
  notSet = () => 'not set'
}

```

**Output:**
```jsx
class GetSetArrows {
  get = () => "get";
  set = () => "set";

  notGet = () => "not get"
  notSet = () => "not set"
}

```

**Expected behavior:**

The property initializers for `get` and `set` should not have semis.

**Comment by duailibe** [comment]
Fixed in 1.9.2!

**Comment by vjeux** [comment]
Nice catch!
